website/static/js/pages/wiki-edit-page.js

```diff
@@ -85,6 +86,15 @@ if (ctx.canEditPageName) {
 
 // Apply panels
 $(document).ready(function () {
+
+    // Treebeard Wiki Menu
+    $.ajax({
```

Comment:
Yep, it's right [here](https://github.com/CenterForOpenScience/osf.io/pull/2990/files#diff-d1d4c751597f63e450870e176e67a0caR36).